OrangeHRMTests/PageObjects/Modules/LeftMenuNavigationPanel.cs

```diff
@@ -0,0 +1,37 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.Drivers;
+using OrangeHRMTests.Common.WebElements;
+using System;
+
+namespace OrangeHRMTests.PageObjects.Modules
+{
+    public class LeftMenuNavigationPanel
+    {
+        private MyWebElement SearchInputField = new MyWebElement(By.XPath("//input[@placeholder='Search']"));
+        private MyWebElement SearchResultElement = new MyWebElement(By.XPath("//span[@class='oxd-text oxd-text--span oxd-main-menu-item--name']"));
+
+        public string LeftMenuItem = "//a[@href='/web/index.php/{0}']";
+
+        public void GoToLeftMenuItem(string value) => new MyWebElement(By.XPath(string.Format(LeftMenuItem, value))).Click();
+
+        public void GoToAdminPage() => GoToLeftMenuItem("admin/viewAdminModule");
```

Comment:
not the best solution because dev may change href value a bit and all your tests will faild
I would better create locator based on text 
![image](https://github.com/agrren/ReneiskiDiploma/assets/55654931/7aee7ab7-884f-4c16-901c-4620b25547d9)

Reply:
Fixed